src/sqlancer/stonedb/StoneDBOptions.java

```diff
@@ -26,6 +26,9 @@ public class StoneDBOptions implements DBMSSpecificOptions<StoneDBOracleFactory>
     public static final String DEFAULT_HOST = "localhost";
     public static final int DEFAULT_PORT = 3306;
 
+    @Parameter(names = "--ignore-logic", description = "Specifies whether or not to compare query result with expectation", arity = 0)
+    public static boolean ignoreLogic;
```

Comment:
You can consider adding a new oracle, which only executes queries without validating results, instead of ignoring existing oracles.

Reply:
ok